### ISSUE TITLE
Add Web Accessibility Insights scan to PR checks

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#6190]
# Platform

* JavaScript

# Issue

Accessibility Insights should run for every PR and report the outcome as a Github check, rather than simply run nightly

**Comment by golddove** [comment]
For now, we merged a daily workflow (see #6069), the results of which we can check at https://github.com/microsoft/AdaptiveCards/actions/workflows/accessibility-insights-daily.yml

Need to follow up with Accessibility Insights team for getting this to work with our CI. Pasting the info here again for reference:

In order for us to use this as a check for every pull request, there still seem to be some difficulties. We currently use Azure Devops pipelines to build our website for every pull-request, and there doesn’t seem to be a way to run the accessibility-insights check after the ADO pipeline finishes execution. These are approaches I’ve tried:

•	Trigger the workflow off of Github’s check_run event (from Events that trigger workflows). I was able to configure this to correctly trigger after the relevant ADO pipeline has completed. However, the pass/fail status of the accessibility insights workflow does not get attached to the commit/PR. The only way to see the results of the run is to look through Github’s Actions tab history of runs.

•	Trigger the workflow regularly (on PR), and use another action (e.g. Wait on check · Actions · GitHub Marketplace) to wait for the ADO pipeline to complete. However, the workflow seems to begin before ADO even has a chance to report that the check is being run, so the Wait on check action can’t find a check to wait on.
 
We do not want to replicate the actual build on ADO and Github Actions (as this would mean maintaining duplicate pipelines), so we need a way to run Accessibility Insights after our ADO build pipeline has completed.

cc: @almedina-ms
